app/models/post.rb

```diff
@@ -1,2 +1,5 @@
 class Post < ApplicationRecord
+  has_many :likes
+  has_many :comments
+  belongs_to :user
```

Comment:
You made use of `author_id` in your post table, hence for consistency make use of author in the association😉.
```suggestion
  belongs_to :author, `class_name: 'User'` 
```